app/models/wifi_user.rb

```diff
@@ -0,0 +1,10 @@
+class WifiUser < ActiveRecord::Base
```

Comment:
This should inheret from `ApplicationRecord` it's a Rails 5 thing.

Or it could just be a PORO that includes `ActiveModel:Model` like the `LogsSearch` class:
```
class LogsSearch
  include ActiveModel::Model
```

Reply:
Good point, I'll update to inherit from `ApplicationRecord` as this will still give me the AR finder methods needed for `find_by` and should ensure the resulting SQL query is clean.